scripts/release/update-comment.js

```diff
@@ -1,27 +1,49 @@
-#!/usr/bin/env node
-
 const fs = require('fs');
-const glob = require('glob');
-const proc = require('child_process');
+const git = require('gift');
+const semver = require('semver');
 
-const gitStatus = proc.execFileSync('git', ['status', '--porcelain', '-uno', 'contracts/**/*.sol']);
```

Comment:
I'm not very happy about adding a new dependency (`gift`). This line shows how to invoke the `git` command and it should be pretty simple and it doesn't involve callbacks. What do you think about using this instead of gift?

Reply:
I know realize that while it as 16441 download/week (which I think is decent) it hasn't been updated in 4 years. I thought that in some cases it would help with parsing the results. But I guess we can drop that dependency and do the parsing ourselves.

What about `semver` ?

Reply:
`semver` is fine!